### PR TITLE
Create Lionbrand.com

### PR DESCRIPTION
**Pull request by zoracon**

### CODE REVIEW

File: src/chrome/content/rules/Lionbrand.com.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!--
+	Refused:
+      Seems like most functioning subdomains do not have https
```
Comment (pipboy96) [inline src/chrome/content/rules/Lionbrand.com.xml]: `s/      /\t\t/`

File: src/chrome/content/rules/Lionbrand.com.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!--
+Refused:
```
Comment (pipboy96) [inline src/chrome/content/rules/Lionbrand.com.xml]: Maybe this line should also have some indentation?

Comment (zoracon) [reply src/chrome/content/rules/Lionbrand.com.xml]: Sure

**Comment by pipboy96** [comment]
You currently have both spaces and tabs in your code. Maybe only one of these should be used?

**Comment by zoracon** [comment]
> You currently have both spaces and tabs in your code. Maybe only one of these should be used?
My code editor is set up with tabs at 2 spaces, Ive only used tabs. @pipboy96,

**Comment by pipboy96** [comment]
@zoracon

**Comment by pipboy96** [comment]
@zoracon LGTM!

**Comment by zoracon** [comment]
> @zoracon LGTM!

Turns out my tabs in my code editor was inserting spaces :/. IDEs.

**Comment by pipboy96** [comment]
@zoracon Try setting up your IDE to use EditorConfig: https://github.com/EFForg/https-everywhere/blob/master/.editorconfig

**Pull request merged**